examples/seismic/elastic/elastic_example.py

```diff
@@ -50,8 +50,8 @@ def run(shape=(50, 50), spacing=(20.0, 20.0), tn=1000.0,
 def test_elastic():
     _, _, _, [rec1, rec2, v, tau] = run()
     norm = lambda x: np.linalg.norm(x.data.reshape(-1))
-    assert np.isclose(norm(rec1), 29.8925, atol=1e-3, rtol=0)
-    assert np.isclose(norm(rec2), 1.20394, atol=1e-3, rtol=0)
+    assert np.isclose(norm(rec1), 29.6567, atol=1e-3, rtol=0)
```

Comment:
I'm assuming this change is dictated by some minor changes you've made to the preset models